demo/demo-jaxrs/jaxrs-client/src/main/java/org/apache/servicecomb/demo/jaxrs/client/JaxrsClient.java

```diff
@@ -147,4 +169,85 @@ private static void testRawJsonParam(RestTemplate template, String cseUrlPrefix)
     TestMgr.check("hello Tom",
         template.postForObject(cseUrlPrefix + "/compute/testrawjson", jsonPerson, String.class));
   }
+
+  private static void testValidatorAddFail(RestTemplate template, String cseUrlPrefix) {
+    Map<String, String> params = new HashMap<>();
+    params.put("a", "5");
+    params.put("b", "3");
+    boolean isExcep = false;
+    try {
+      template.postForObject(cseUrlPrefix + "add", params, Integer.class);
+    } catch (InvocationException e) {
+      isExcep = true;
+      TestMgr.check(400, e.getStatus().getStatusCode());
+      TestMgr.check(Status.BAD_REQUEST, e.getReasonPhrase());
+      TestMgr.check(
+          "CommonExceptionData [message=[ConstraintViolationImpl{interpolatedMessage='must be greater than or equal to 20', propertyPath=add.arg1, rootBeanClass=class org.apache.servicecomb.demo.jaxrs.server.Validator, messageTemplate='{javax.validation.constraints.Min.message}'}]]",
```

Comment:
so strange that use exception.toString to be the message.
infact only interpolatedMessage/propertyPath/rootBeanClass is useful
we can use json to express this.

Reply:
I'll submit a new PR for this minor problem in https://github.com/apache/incubator-servicecomb-java-chassis/pull/649. And merge this one.

Reply:
it's not about test case......
root cause is that developer use exception.toString to be response information.